lib/quickbooks/model/report.rb

```diff
@@ -112,7 +112,7 @@ def zip_rows
             @columns.each_with_index do |column, col_idx|
               next if row_data[col_idx].nil?
 
-              row_data[col_idx].merge!(column.transform_keys { |col_key| col_key.to_s.tr('col_', '').to_sym })
+              row_data[col_idx].merge!(column.transform_keys { |col_key| col_key.to_s.gsub('col_', '').to_sym })
```

Comment:
I think if you were to add `rubocop`, it would complain about this
```suggestion
              row_data[col_idx].merge!(column.transform_keys { |col_key| col_key.to_s.delete('col_').to_sym })
```

Reply:
Just tested and that breaks the keys.

For example:

```ruby
rows = [{ index: 0, columns: [{ title: 'Date', ... }], ... }]

# Transform keys
[{ index: 0, columns: [{ tite: 'Date', ... }], ... }]
```

Reply:
Ah, rubocop probably complains about `gsub!` then, my bad.

Reply:
> Ah, rubocop probably complains about `gsub!` then, my bad.

No worries. I had `tr`, but it was causing the same issue. Then you said to use `delete` and I forgot about that, so I just made the change, but then I read the docs and it works almost the same as `tr` in this case.

I'm not sure if I'll ever add rubocop because a lot of formatting would have to happen (a lot might be covered automatically?).